keycloak/data/havendev-realm.json

```diff
@@ -751,13 +751,13 @@
   }, {
     "id" : "02f6b754-9ed8-4254-adbd-8a1f845b1ac7",
     "clientId" : "havendev",
-    "rootUrl" : "http://localhost:2015/",
+    "rootUrl" : "",
     "adminUrl" : "http://localhost:2015/",
     "surrogateAuthRequired" : false,
     "enabled" : true,
     "clientAuthenticatorType" : "client-secret",
     "secret" : "d6c6ec35-c755-41d1-837d-e4e9eed4cb1a",
-    "redirectUris" : [ "http://localhost:2015/*", "https://emurphy-haven.ngrok.io/*" ],
+    "redirectUris" : [ "https://emurphy-haven.ngrok.io/*", "/*" ],
```

Comment:
also this line. the rest of the changes in this file are just re-ordering that happened during the keycloak export of the havendev realm.